util/design/lib/OtpMemMap.py

```diff
@@ -8,10 +8,11 @@
 
 import logging as log
 import random
+from collections import OrderedDict
```

Comment:
Not that it matters, but I'd expect this code to work with just dicts now (we require Python 3.6, where [dicts iterate in the order in which elements were added](https://stackoverflow.com/questions/39980323/are-dictionaries-ordered-in-python-3-6)).

Reply:
Ah right. I've done this out of habit to stay on the safe side.
Let's remove this then.